### DISCUSSION TITLE
Code navigation using CTRL+click on constants, functions

### DISCUSSION THREAD

**Discussion by DanieleBenedettelli** [pybricks/support#1061] [Ideas]
As the subject says, I am asking if it would be possible to add to the IDE a code navigation feature, to 
Go to Definition, Go to Implementation, Go to Symbol using CTRL+click.
I don't know how hard is it to implement, or if it's already there and I can't use it.

Thanks!
D

**Comment by dlech** [comment]
If you use VS Code, you can do all of this already. 

For Pybricks Code, we would have to implement the glue between the Jedi language server and the Monaco editor. And we would probably have to make some kind of read-only editor for the modules in the firmware.

**Comment by DanieleBenedettelli** [comment]
Sorry, I forgot to specify that I meant the PyBricks web IDE.
